phpunit.xml

```diff
@@ -6,8 +6,7 @@
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="true">
+         processIsolation="false">
```

Comment:
It's hard to debug multiple tests with it. I can revert it if you against this change

Reply:
It's okay, let's leave it as you suggested.